src/Composer/Installers/CodeIgniterInstaller.php

```diff
@@ -1,11 +1,70 @@
 <?php
 namespace Composer\Installers;
 
+use Composer\Semver\Constraint\Constraint;
+
 class CodeIgniterInstaller extends BaseInstaller
 {
+
     protected $locations = array(
         'library'     => 'application/libraries/{$name}/',
+        'helper' => 'application/helpers/{$name}/',
+        'model' => 'application/models/{$name}/',
         'third-party' => 'application/third_party/{$name}/',
         'module'      => 'application/modules/{$name}/',
     );
+
+    public function inflectPackageVars($vars)
+    {
+        if ($this->matchesVersion('>=', '3.11.1')) {
```

Comment:
Hmm... Why used non-existed version? I don't see this version on releases page: https://github.com/bcit-ci/CodeIgniter/releases.

Reply:
This is the why
![image](https://user-images.githubusercontent.com/3001652/120982596-13c1d980-c7a3-11eb-85d8-755a5393a203.png)

Reply:
The current version is 3.1.11, but you comparing with 3.11 version. It's error?